### DISCUSSION TITLE
Private key from account on local account file

### DISCUSSION THREAD

**Discussion by larsivi** [go-acme/lego#1692] [Q&A] [answered]
Hi,

I have a Lets Encrypt account, where I have created and renewed some certificates manually in the past. I want to automate the process in the cloud, and plan to write a Cloud Function using LEGO.

I see that I in /etc/letsencrypt/accounts/acme-v02.api.letsencrypt.org/directory/ have a file that ends in private_key.json. Does this file, or anything in it, map to the private key used in the main library example (where a new user is created, rather than logging on an existing one)?

**Comment by ldez** [comment, marked as answer]
> where a new user is created, rather than logging on an existing one

Accounts don't really exist with Let's Encrypt: there is no "logging".

The mail and the private key are the way to identify a user, nothing more.

`private_key.json` contains the private key.

I'm not sure about what exactly you are looking for.

**Reply by larsivi** [reply]
I am sorry for being slightly newbie to Go and certain deeper Lets Encrypt concepts :)

I guess I wonder if the full contents of the json-file equates the key, or if it is one of the properties inside.

**Reply by ldez** [reply]
The content of the file is the full key.

**Comment by viharm** [comment]
I have the same question, but will present it slightly differently. I have used certbot in the past, then migrated by account key to a different client, and now have a PEM account key. I am aware that LE does not have accounts, but would still prefer to use the same key. Is it possible in LeGo to provide an existing account key?
